ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -315,17 +315,36 @@ TFLiteInterpreter::getTensorType (TfLiteType tfType)
   switch (tfType) {
     case kTfLiteFloat32:
       return _NNS_FLOAT32;
+
     case kTfLiteUInt8:
       return _NNS_UINT8;
+
     case kTfLiteInt32:
       return _NNS_INT32;
+
     case kTfLiteBool:
+#ifdef TFLITE_INT8
     case kTfLiteInt8:
+#endif
       return _NNS_INT8;
+
+#ifdef TFLITE_INT16
+    case kTfLiteInt16:
+      return _NNS_INT16;
+#endif
+
     case kTfLiteInt64:
       return _NNS_INT64;
+
     case kTfLiteString:
+#ifdef TFLITE_COMPLEX64
+    case kTfLiteComplex64:
+#endif
+#ifdef TFLITE_FLOAT16
+    case kTfLiteFloat16:
+#endif
     default:
+      g_critical ("Not supported data type!");
```

Comment:
Please print out the int-number of the type.

Reply:
it's added